.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "scripts/zfs-images"]
 	path = scripts/zfs-images
 	url = https://github.com/zfsonlinux/zfs-images
+[submodule "man/build/jinja2"]
+	path = man/build/jinja2
+	url = https://github.com/pallets/jinja.git
+	branch = 2.10.x
```

Comment:
> Added jinja2 python module as git submodule so we freeze what version doc builds depend on

I'm just learning about jinja2, but I wonder if freezing the version this way is really needed.  It looks like distribution provided packages are available for Ubuntu, Fedora, Debian, RHEL, and presumably others.  It would be nice if we could instead rely on those packages, installing `python3-jinja2` worked great for me.

If we're concerned about future incompatibilities or availability on some platforms.  Then we could include the submodule and write the scripts to prefer it by default when it has been manually cloned.  That would make it easy to use when otherwise unavailable, which would be nice.

We can add `jinji2` as a dependency installed by the CI to verify it is available on all the supported platforms and works as intended.

Reply:
Yes, agreed. It'd be nice to just use the distro packages, if at all possible.

Reply:
Agreed. Should it be listed as a requirement in `README.md` and distro-specific package names added to `zfs.spec.in` for the build phase? (I'm not intimately familiar with RPM and DEB package building, so correct away!)